src/main/resources/migrations.xml

```diff
@@ -2336,4 +2336,25 @@
 		<dropColumn tableName="tasks" columnName="assessment" />
 	</changeSet>
 
+	<changeSet id="add_task_positions" author="maradragan">
+		<createTable tableName="taskPositions">
```

Comment:
I suggest to have a convention that includes the relationship (`responsible` in this case) in the table name.